PSRabbitMq/Send-RabbitMqMessage.ps1

```diff
@@ -63,21 +102,51 @@
         [string]$Key,
 
         [Parameter(Mandatory=$true,ValueFromPipeline=$true)]
+        [Alias('Payload')]
         $InputObject,
 
         [switch]$Persistent,
 
         [Int32]$Depth = 2,
 
-        [PSCredential]$Credential,
+        [PSCredential]
+        [System.Management.Automation.Credential()]
+        $Credential,
 
         [System.Security.Authentication.SslProtocols]$Ssl,
 
-        [parameter(Mandatory = $false)]
-        [string]$vhost
+        [string]$vhost = '/',
+
+        [ValidateSet('application/clixml+xml','application/json','text/xml', 'text/plain')]
```

Comment:
I know these are the valid content types for this module, but if we're sending a message to another type of consumer it could support another content type, so I'm not sure we want to restrict this here.